src/EnergyPlus/InputProcessing/InputProcessor.hh

```diff
@@ -127,6 +127,14 @@ public:
 
     bool getDefaultValue(EnergyPlusData &state, std::string const &objectWord, std::string const &fieldName, std::string &value);
 
+    std::string getAlphaFieldValue(
+        EnergyPlusData &state, std::string const &objectWord, json const &ep_object, json schema_obj_props, std::string const &fieldName);
+
+    Real64 getRealFieldValue(
+        EnergyPlusData &state, std::string const &objectWord, json const &ep_object, json schema_obj_props, std::string const &fieldName);
```

Comment:
So schema_obj_props is a copy of the original. Should also be referenced?

Reply:
Yes, good catch. And `const`.